### ISSUE TITLE
cex_label_category in emapplot gone

### ISSUE THREAD

**Issue by Mitmischer** [YuLab-SMU/enrichplot#322]
I'm using enrichplot 1.28.2 (according to `BiocManager::valid()` ).

In earlier versions, one could set the label size in `emapplot` via `cex_label_category`.

This options seems to be gone now. I did not find a mention of "cex_label_category" in the changelog (https://github.com/YuLab-SMU/enrichplot/blob/devel/NEWS.md), however there were changes to `emapplot`.
As there was a breaking change, I would expect to find this information somewhere in the changelog.

Also, I can still find mention of the parameter online (https://rdrr.io/bioc/enrichplot/man/emapplot.html), but - rightfully so - not in my local installation. I don't know who maintains the online documentation, but the last change was on 2021 (according to the image at the bottom on the page). This documentation should be updated or removed.

Anyway - in the outdated documentation I found declarations of parameter renaming (node_label_size - size of node label, this parameter has been changed to cex_label_category). That shows that renaming has been well handled in the past.

Of course, the issue at hand is a minor issue. Ultimately and more generally, I want to make suggestions about clearer communication of breaking changes (in the changelog _and_ in the function itself (for example, consider the way sklearn documents their changes: https://scikit-learn.org/stable/modules/generated/sklearn.discriminant_analysis.LinearDiscriminantAnalysis.html), finer commit messages with a more thorough description of changes and consistency of online documentation to make such cases easier to handle in the future.

Lastly, which parameter replaces `cex_label_category`? `size_category`?

**Comment by DanaiNM** [comment]
i also have this problem. whatever i do, i can seem to increase the font size of the category and the gene names...

**Comment by cstubben** [comment]
I had to use ggtangle and geom_cnet_label to pass the font size to geom_text_repel following issue [302](https://github.com/YuLab-SMU/enrichplot/issues/302) 
emapplot(x, showCategory = 40, min_edge=0.5, node_label="none") + geom_cnet_label(node_label = "all",  size = 3)

**Comment by JakeLehle** [comment]
@cstubben Thank you!

Here was the code I ran to fix this issue.
```
library(enrichplot)
library(ggtangle)

(other code ...)

p <- emapplot(pairwise_termsim(go_enrich), showCategory = 10, size_category = 2.5, node_label = "none") + 
          geom_cnet_label(node_label = "all", size = 9)
```

**Comment by GuangchuangYu** [comment]
Fixed, and this duplicates #307: `emapplot()` again supports controlling category label size, via the `node_label_size` argument (2b93e5b), documented in the `.Rd`. Closing as a duplicate of #307.
